hq_superset/utils.py

```diff
@@ -415,3 +418,7 @@ def cast_data_for_table(
 def generate_secret():
     alphabet = string.ascii_letters + string.digits
     return ''.join(secrets.choice(alphabet) for __ in range(64))
+
+
+def datetime_utcnow_naive():
```

Comment:
@mkangia 
I think this function name is a misnomer. You're returning an [aware datetime](https://docs.python.org/3/library/datetime.html#aware-and-naive-objects), are you not?.

Reply:
You are right, I got that backwards, will rename this `datetime_utcnow`.